File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/utils/SqlUtils.java
package com.skedgo.tripkit.ui.utils;

import android.database.sqlite.SQLiteStatement;

class SqlUtils {
    public static void bind(SQLiteStatement statement, int bindArg, String value) {
        if (value == null) {
            statement.bindNull(bindArg);
        } else {
            statement.bindString(bindArg, value);
        }
    }
}

File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/utils/SqlUtils.kt
package com.skedgo.tripkit.ui.utils

import android.database.sqlite.SQLiteStatement

object SqlUtils {
    fun bind(statement: SQLiteStatement, bindArg: Int, value: String?) {
        if (value == null) {
            statement.bindNull(bindArg)
        } else {
            statement.bindString(bindArg, value)
        }
    }
}